docs/source/download.rst

```diff
@@ -1,111 +1,200 @@
-=============
-Obtain Devito
-=============
+===========================
+Download and Install Devito
+===========================
+
+There are two main approaches to installing Devito.
+
+- `Docker installation`_, for those looking for a quick “out-of-the-box” way to try Devito
+- `pip/conda installation`_, for those looking to use Devito as part of a project alongside other packages or/and looking to develop with Devito
```

Comment:
"or/and looking to develop with Devito" lots of repeated words

Reply:
not sure what to do here, what would you propose?

Reply:
keep master's ?

Reply:
not sure, in master we actually split pip/conda in two for actually no reason in my opinion

Reply:
I would drop:

```
or/and looking to develop with Devito
```

Reply:
Docker installation -> Docker

pip/conda installation -> Virtual environment